src/components/DateRange/i18n/ru.json

```diff
@@ -0,0 +1,4 @@
+{
+  "date-format": "DD.MM.YYYY",
```

Comment:
We decided not to translate application to Russian. So, it's no need to add RU keysets.
And if you are editing keysets where ru presents, please remove it.

Reply:
fixed